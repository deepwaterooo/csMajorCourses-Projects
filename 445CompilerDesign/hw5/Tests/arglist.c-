// C-F13
// C-F09
int kate(int is; bool bs; int ia[]; bool ba[])
{
	return 666;
}

int matty()
{
    int inta[2];
    bool boola[3];

    kate(103+101, true, inta, boola);

    return 42;
}
