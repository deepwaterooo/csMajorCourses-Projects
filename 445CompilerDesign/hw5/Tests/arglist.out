ERROR(13): Expecting type bool in parameter 3 of call to 'kate' defined on line 3 but got type int.
ERROR(13): Too many parameters passed for function 'kate' defined on line 3.
ERROR(LINKER): Procedure main is not defined.
Number of warnings: 0
Number of errors: 3
------------------------------------

Memory Configuration: Data Addresses: 0-9999  Instruction Addresses: 0-9999
Loading file: arglist.tm
Halted
Bye.
